### PR TITLE
Add NetworkProbe to switch case expression.

### PR DESCRIPTION
**Pull request by taras-skycoin**
Did you run `make format && make check`?
yes
Fixes https://github.com/SkycoinPro/skywire-services/issues/316

 Changes:	
-	Allow to route NetworkProbePackat

How to test this PR:
You need to set up a test environment.

### CODE REVIEW

**Review by jdknives — CHANGES_REQUESTED** [review]
This change alone won't do. What happens is that Network Probe packets get forwarded but the remote visor identifies them as data packets and therefore forwards them to the application using the route. This causes problems. 

Bildschirmfoto 2020-11-07 um 14 53 08

**Review by jdknives — CHANGES_REQUESTED** [review]
Good job. This fixed the issues with `NetworkProbe` packets being forwarded to applications. Before we merge, if this PR is related to this issue 

Bildschirmfoto 2020-11-10 um 13 59 44

Please try to reproduce this warning on Visor B in integration env. @taras-skycoin

**Review by jdknives — APPROVED** [review]
Just checked and the error is not related to your PR. Can be merged after one more review.

**Review by nkryuchkov — APPROVED** [review]

**Pull request merged**
